fleet_vehicle_inspection/security/fleet_vehicle_inspection.xml

```diff
@@ -24,4 +24,48 @@
         <field name="perm_unlink" eval="1"/>
     </record>
 
+    <!-- Record Rules -->
+
+    <record id="fleet_rule_inspection_visibility_user" model="ir.rule">
+        <field name="name">User can only see his/her inspection</field>
+        <field name="model_id" ref="model_fleet_vehicle_inspection"/>
+        <field name="groups" eval="[(4, ref('fleet.fleet_group_user'))]"/>
+        <field name="perm_read" eval="1"/>
+        <field name="perm_write" eval="0"/>
+        <field name="perm_create" eval="0"/>
+        <field name="perm_unlink" eval="0"/>
+        <field name="domain_force">[('vehicle_id.driver_id','=',user.partner_id.id)]</field>
+    </record>
```

Comment:
The purpose of this rule is not clear to me, I understand like the drivers of the inspected vehicles can enter the system and see only the inspections that have been made to their vehicles ... Is that so?

Reply:
Yes. I thought of exactly that.